components/Service/ServicePage.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable react/jsx-indent-props */
+import React from 'react';
+import Head from 'next/head';
+import NavigationBar from '../Common/NavigationBar';
+import './service.scss';
+import { HOME_STRINGS } from '../constants';
+import { SERVICE_STRING } from './constants';
+
+const {
+    GOOGLE_FONTS,
+    INTRO_TEXT2,
+} = SERVICE_STRING;
+
+const {
+    FONT_AWESOME,
+    INTEGRITY,
+    LIFE,
+    HELP_DESK,
+    SERVICES,
+    CONSULT_TEXT1,
+    CONSULT_TEXT2,
+    READ_MORE,
+} = HOME_STRINGS;
+
+const ServicePage = () => {
+    return (
+        <React.Fragment>
+            <Head>
+                <link href={GOOGLE_FONTS} />
+                <link
+                    rel="stylesheet"
+                    href={FONT_AWESOME}
+                    integrity={INTEGRITY}
+                    crossOrigin="anonymous"
+                />
+            </Head>
+            <div className="header">
+                <NavigationBar />
+                <div className="header-cover">
+                    <span className="header-cover-text">
+                        <p className="text1">
+                            {LIFE}
+                            {' '}
+                        </p>
+                        {' '}
+                        <p className="text2">Services</p>
+                    </span>
+                    <p className="desc">Home  /  Services</p>
+                </div>
+            </div>
+            <div className="intro app-content">
+                <hr />
+                <p className="intro-header">{SERVICES}</p>
+                <div className="intro-service-wrapper">
+                    <div className="content-wrapper services-wrapper intro-wrap">
+                        <div className="intro-content">
+                            <i className="fas fa-users icons-status" />
+                            <hr />
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                   Online Counselling
```

Comment:
move strings to constant

Reply:
I don't see the need to move a non-reusable string that is less than 20 characters into a constant file. That's why you can see those few strings around